ee/clickhouse/queries/breakdown_props.py

```diff
@@ -1,25 +1,25 @@
 from typing import Any, Dict, List, Optional, Tuple, Union, cast
 
-from ee.clickhouse.client import sync_execute
 from ee.clickhouse.models.cohort import format_filter_query
 from ee.clickhouse.models.entity import get_entity_filtering_params
 from ee.clickhouse.models.property import (
     get_property_string_expr,
     get_single_or_multi_property_string_expr,
     parse_prop_grouped_clauses,
 )
-from ee.clickhouse.models.util import PersonPropertiesMode
-from ee.clickhouse.queries.column_optimizer import ColumnOptimizer
+from ee.clickhouse.queries.column_optimizer import EE_ColumnOptimizer
```

Comment:
I assume EE_ColumnOptimizer will be gone at the end of all these PRs?

Reply:
Nope, to adhere to the principle of FOSS, keeping as much of the group logic in the /ee directory as possible

Reply:
Ah, I see, right, makes sense!

Can I propose renaming all of these to:

`EnterpriseColumnOptimizer`, `EnterpriseEventQuery` , etc. etc.